### ISSUE TITLE
App crashes on startup when using certain Nextcloud theme colors

### ISSUE THREAD

**Issue by bubelov** [stefan-niedermann/nextcloud-deck#466]
**Describe the bug**

My Nextcloud returns account.color = #000, which [Android's Color class can't parse](https://developer.android.com/reference/android/graphics/Color.html#parseColor%28java.lang.String%29) so the app crashes on this line:

```java
Application.saveBrandColors(this, Color.parseColor(mainViewModel.getCurrentAccount().getColor()), Color.parseColor(mainViewModel.getCurrentAccount().getTextColor()));
```

**To Reproduce**
1. Go to Nextcloud -> Settings -> Theming
2. Set color as #000000
3. Install Nextcloud Deck
4. Try to sign in

So, that's a bit tricky. The app assumes that colors are always 6 hex chars long but Nextcloud seem to "shorten" theme colors to just 3 hex chars under certain conditions. I'm not familiar with the Nextcloud API so I'm not sure whether it's their issue or not. Anyway, it seems like there are lots of Nextcloud instances that can return such values so the app shouldn't crash on them.

I can submit a path if the proposed way of solving this issue is OK with the maintainers.

**Comment by stefan-niedermann** [comment]
Absolutel, thank you very much for this very detailled bug report! Will fix as soon as possible.

**Comment by stefan-niedermann** [comment]
I'll do this one.

**Comment by stefan-niedermann** [comment]
Will be fixed in version `1.1.1`. Thanks!
